lib/move/__tests__/move-preserve-timestamp.test.js

```diff
@@ -0,0 +1,97 @@
+'use strict'
+
+const fs = require('../../')
+const os = require('os')
+const path = require('path')
+const utimesSync = require('../../util/utimes').utimesMillisSync
+const assert = require('assert')
+const fse = require('../../index')
+
+/* global beforeEach, afterEach, describe, it */
+
+if (process.arch === 'ia32') console.warn('32 bit arch; skipping copy timestamp tests')
+
+const describeIfPractical = process.arch === 'ia32' ? describe.skip : describe
+
+describeIfPractical('move() - across different devices', () => {
+  let TEST_DIR, SRC, DEST, FILES
+  let __skipTests = false
+  const differentDevicePath = '/mnt'
+
+  // must set this up, if not, exit silently
+  if (!fs.existsSync(differentDevicePath)) {
+    console.log('Skipping cross-device move test')
+    __skipTests = true
+  }
+
+  // make sure we have permission on device
+  try {
+    fs.writeFileSync(path.join(differentDevicePath, 'file'), 'hi')
+  } catch {
+    console.log("Can't write to device. Skipping move test.")
+    __skipTests = true
+  }
+
+  function setupFixture (readonly) {
+    TEST_DIR = path.join(os.tmpdir(), 'fs-extra', 'move-sync-preserve-timestamp')
+    SRC = path.join(differentDevicePath, 'some/weird/dir-really-weird')
+    DEST = path.join(TEST_DIR, 'dest')
+    FILES = ['a-file', path.join('a-folder', 'another-file'), path.join('a-folder', 'another-folder', 'file3')]
+    const timestamp = Date.now() / 1000 - 5
+    FILES.forEach(f => {
+      const filePath = path.join(SRC, f)
+      fs.ensureFileSync(filePath)
+      // rewind timestamps to make sure that coarser OS timestamp resolution
+      // does not alter results
+      utimesSync(filePath, timestamp, timestamp)
+      if (readonly) {
+        fs.chmodSync(filePath, 0o444)
+      }
+    })
+  }
+
+  const _it = __skipTests ? it.skip : it
+
+  afterEach(done => {
+    fse.removeSync(TEST_DIR)
+    fse.removeSync(SRC)
+    done()
```

Comment:
Don't need `done` for synchronous hooks
```suggestion
  afterEach(() => {
    fse.removeSync(TEST_DIR)
    fse.removeSync(SRC)
```